src/demo/examples/visualization-bar/index.js

```diff
@@ -0,0 +1,90 @@
+
+import { visualizationBar, VisualizationBar, VisualizationBarSizes } from 'hexagon-js';
+import { div } from 'utils/selection';
+
+const randNum = max => Math.round(Math.random() * max);
+const rand = max => e => {
+  e.progress(randNum(max));
+};
+
+export default () => {
+  const selectionWithMax = div();
+  const selectionErrored = div();
+  const selectionWithoutMax = div();
+  const selectionWithContent = div();
+  const selectionWithLabels = div();
+  const selectionSmall = div();
+
+  const mocks = [
+    [new VisualizationBar(selectionWithMax.node(), {
+      title: 'With maximum value', buffer: 2, value: 5, max: 20,
+    }), rand(20)],
```

Comment:
If you're wrapping these in arrays, could you format it like:

```js
[
  new VisualizationBar(selectionWithMax.node(), {
    title: 'With maximum value', buffer: 2, value: 5, max: 20,
  }), 
  rand(20),
]
```

Makes it much easier to read. Without this, it's difficult to notice that `rand(20)` isn't part of the `VisualizationBar` construction)